content/ch-quantum-hardware/cQED-JC-SW.ipynb

```diff
@@ -469,7 +469,7 @@
    "source": [
     "This is typically written as\n",
     "$$\n",
-    "H_{\\rm eff} = \\left(\\omega_r + \\frac{g^2}{\\Delta}\\sigma_z\\right)a^\\dagger a  - \\frac{1}{2}\\left(\\omega_q -\\frac{g^2}{\\Delta}\\right) \\sigma_z\n",
+    "H_{\\rm eff} = \\frac{g^2}{2\\Delta} \\left(\\omega_r + \\frac{g^2}{\\Delta}\\sigma_z\\right)a^\\dagger a  - \\frac{1}{2}\\left(\\omega_q -\\frac{g^2}{\\Delta}\\right) \\sigma_z\n",
```

Comment:
I think the original version is correct here.

Reply:
Sorry forgot to take a screenshot. I think in the second equation the 1 in the brakets of the first equation is missing (clearly in eq1 are 5 terms, but in eq2 just 4).
![6_1_missing_term](https://user-images.githubusercontent.com/43704364/96908160-88f2cf80-149c-11eb-8edf-1aa8cee12bfe.png)